scene/resources/animation.cpp

```diff
@@ -3123,6 +3123,8 @@ void Animation::track_get_key_indices_in_range(int p_track, double p_time, doubl
 				from_time += CMP_EPSILON;
 			}
 		} break;
+		default: {
+		} break;
```

Comment:
I see, so the challenge with combining LoopMode enum into Animation and AnimationNodeAnimation is that the extra enum value may be possible to set in loop_mode here, and the unused case statement may be needed.

Should the default case be the same as `LOOP_NONE`?

Reply:
> Should the default case be the same as LOOP_NONE?

It will break compatibility. Also, it should not be overridden by default. So `No Override` should be the default for AnimationNodeAnimation.